lib/appmap/agent.rb

```diff
@@ -100,5 +100,9 @@ def detect_metadata
       @metadata ||= Metadata.detect.freeze
       Util.deep_dup(@metadata)
     end
+
+    def parameter_schema?
+      ENV['APPMAP_PARAMETER_SCHEMA'] == 'true'
```

Comment:
Perhaps this could be memoized to avoid repeated string comparisons (although, granted, most of the time it'd just be a `nil` so I suppose it's not that important).